CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+<a name="1.114.5"></a>
+## 1.114.5 (2018-06-22)
+
+
```

Comment:
I believe this is empty because the change log ignores merge branches.

Reply:
Can you add a bit of text that says this merges v1.113.5 into master?